tests/test_core.py

```diff
@@ -5910,7 +5910,7 @@ def test_unistd_unlink(self, fs):
       if WINDOWS:
         self.emcc_args += ['-DNO_SYMLINK=1']
       if MACOS:
-        self.skipTest()
```

Comment:
Hmm .. shame we don't have something more useful to write here.. the old code didn't document the reason.